### PR TITLE
Added to the Search in Aggregate

### PR DESCRIPTION
**Pull request by ifeLawal**
Added 3 new general job boards and the process for specific job boards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,22 @@ As a bonus, these modern job boards can be easily searched by Google, and *filte
 
 :mag_right: [View all companies posting to **Lever** (Google Search)](https://www.google.com/search?hl=en&q=site%3Ahttps%3A%2F%2Fjobs.lever.co "View all companies posting to Lever (Google Search)")
 
+:mag_right: [View all companies posting to **Workday** (Google Search)](https://www.google.com/search?hl=en&q=site%3Ahttps%3A%2F%2Fwd5.myworkdayjobs.com "View all companies posting to Workday (Google Search)")
```
Comment (hdehal) [inline README.md]:
```suggestion
:mag_right: [View all companies posting to **Workday** (Google Search)](https://www.google.com/search?hl=en&q=site%3Ahttps%3A%2F%2F*.myworkdayjobs.com "View all companies posting to Workday (Google Search)")
```

Comment (hdehal) [reply README.md]: Thanks for adding these!

Re: The Workday search, I did a quick test and wasn't sure the difference between the wd5, wd4, wd3, wd2, etc. subdomains, e.g. from https://wd5.myworkdayjobs.com -- however adding a wildcard "*" seems to search all of the domains at once!

Comment (ifeLawal) [reply README.md]: Oh nice! I didn't even notice they had multiple subdomains, so that wildcard is awesome!

**Comment by hdehal** [comment]
Thanks @ifeLawal for the pull request and adding to the list!

**Pull request merged**